.tekton/operator-bundle-build.yaml

```diff
@@ -16,15 +17,17 @@ metadata:
     pipelines.appstudio.openshift.io/type: build
   name: operator-bundle-build
   namespace: rh-acs-tenant
+
 spec:
+
   params:
   - name: dockerfile
-    value: operator/bundle.Dockerfile.konflux
+    value: operator/konflux.bundle.Dockerfile
   - name: git-url
-    value: '{{source_url}}'
+    value: '{{repo_url}}'
   # TODO(ROX-24530): return expiration for non-released images to 13w
   - name: image-expires-after
-    value: 5d
+    value: '52w'
```

Comment:
I'd say we should keep it at 5d for now until we have a working bundle index. Until then we have no real way of making sure the bundles we're producing are actually usable.

Reply:
There's **no** need to optimize for storage for the service we don't pay for and for images that are so tiny.
There's no harm in producing these images because image tags are unique for each commit.
The disadvantage of having `5d` is that it's not explained anywhere and so creates a maintenance burden: each time anyone looks at it wanting to change, the person would wonder why is it `5d` here and `'52w'` everywhere else.